third_party/move/extensions/move-table-extension/Cargo.toml

```diff
@@ -11,16 +11,16 @@ publish = false
 [dependencies]
 better_any = { workspace = true }
 bytes = { workspace = true }
-move-binary-format = { path = "../../move-binary-format" }
-move-core-types = { path = "../../move-core/types" }
-move-vm-runtime = { path = "../../move-vm/runtime" }
-move-vm-types = { path = "../../move-vm/types" }
+move-binary-format = { workspace = true }
+move-core-types = { workspace = true }
+move-vm-runtime = { workspace = true }
+move-vm-types = { workspace = true }
 sha3 = { workspace = true }
 smallvec = { workspace = true }
 
 [dev-dependencies]
-move-cli = { path = "../../tools/move-cli" }
-move-package = { path = "../../tools/move-package" }
+move-cli = { workspace = true }
+move-package = { workspace = true }
 move-stdlib = { path = "../../move-stdlib", features = ["testing"] }
```

Comment:
Could be made `{ workspace = true }`?

Reply:
For stdlib, I am not sure mostly because we have 2 of those😄 one we use for Aptos (as a workspace dependency `aptos-move-stdlib`), and one defined in third-party move. I think the reason for this is gas, and we want to move some of gas infra (cc @vgao1996 to confirm my thinking) to third party to be able to use it directly. I did not add it as a workspace dependency to avoid confusion between the two.